### DISCUSSION TITLE
Add support for Photo Sphere (360°) panoramas

### DISCUSSION THREAD

**Discussion by DecaTec** [mediacms-io/mediacms#623] [Feature Requests]
**Describe the feature you'd like**
I often create 360° panoramas. When uploading these to MediaCMS, these are not shown as Photo Sphere Panoramas, but as "flat" images.

You probably know Photo Sphere from Google (maps), these are 360° panoramas, you can take a look at an example [here](https://photo-sphere-viewer.js.org/playground.html).

**Additional context**
There are a few libraries out there, e.g. [Photo Sphere Viewer](https://photo-sphere-viewer.js.org/) (or [GitHub](https://github.com/mistic100/Photo-Sphere-Viewer)). Maybe these could be integrated into MediaCMS.

**Comment by mgogoulos** [comment]
Worth to explore!

**Comment by hemel-cse** [comment]
+1
